src/routes/grantform/index.js

```diff
@@ -17,7 +17,13 @@ export default [
   {
     path: '/grant/success',
 
-    async action() {
+    async action({ store }) {
+      const { submission } = store.getState();
+
+      if (!submission) {
+        return { redirect: '/' };
+      }
```

Comment:
Ok nice. So each action here has access to the store. This is great. I was imagining that this was possible but couldn't see an example anywhere.